src/text/story.tsx

```diff
@@ -4,17 +4,29 @@ import { storiesOf } from '@storybook/react'
 import { withInfo } from '@storybook/addon-info'
 import { selectV2, text, withKnobs, boolean } from '@storybook/addon-knobs'
 
+import { color } from '_utils/branding'
 import Text, { TextTagType, TextDisplayType } from 'text'
 
 const stories = storiesOf('Text', module)
 stories.addDecorator(withKnobs)
 
+const c = Object.keys(color).reduce(
```

Comment:
To remove ?

Reply:
Ah ok it's used above. Can you change the name to make it more explicit ?

Reply:
We agree that this code doesn't do anything ... Or I didn't get it

Reply:
Oh yes... the result is... `color`

Reply:
I commited the change